package.json

```diff
@@ -27,6 +27,7 @@
     "lodash-compat": "^3.9.3",
     "object.assign": "^2.0.1",
     "safe-json-parse": "^4.0.0",
+    "videojs-font": "1.1.0",
```

Comment:
Did you mean to make this a hard requirement? We should relax this so it will pull the most recent version.